packages/graph-explorer-proxy-server/node-server.js

```diff
@@ -147,302 +150,300 @@ async function fetchData(res, next, url, options, isIamEnabled, region, serviceT
       serviceType
     );
     const data = await response.json();
+    res.status(response.status || 200);
```

Comment:
could empty response and 204 be here?

Reply:
After looking in to it a bit I ended up removing the `|| 200` fallback.

The `response.status` value should never be undefined. I was following the pattern I saw with error flow. Since this `response` object is coming directly from `fetch()` it should always have a `status`.